df.viewscreen.xml

```diff
@@ -1629,11 +1629,11 @@
         </stl-vector>
         <stl-vector type-name='int32_t' name='gametype'/>
         <stl-vector name='gametype_str'/>
```

Comment:
`gametype_str` is a vector of `stl-string`

Reply:
can't figure out how to get this one populated, but I'll add that too